userspace/falco/app/actions/helpers_inspector.cpp

```diff
@@ -61,9 +61,15 @@ falco::app::run_result falco::app::actions::open_live_inspector(falco::app::stat
 					falco_logger::log(
 					        falco_logger::level::INFO,
 					        "Opening '" + source + "' source with plugin '" + cfg->m_name + "'");
-					inspector->open_plugin(cfg->m_name,
-					                       cfg->m_open_params,
-					                       sinsp_plugin_platform::SINSP_PLATFORM_HOSTINFO);
+					if(s.config->m_plugins_hostinfo) {
+						inspector->open_plugin(cfg->m_name,
+						                       cfg->m_open_params,
+						                       sinsp_plugin_platform::SINSP_PLATFORM_HOSTINFO);
+					} else {
+						inspector->open_plugin(cfg->m_name,
+						                       cfg->m_open_params,
+						                       sinsp_plugin_platform::SINSP_PLATFORM_GENERIC);
+					}
```

Comment:
While it simple it is a lot of repetition for switching a parameter.
I would suggest something a bit simpler:
```suggestion
					sinsp_plugin_platform platform = sinsp_plugin_platform::SINSP_PLATFORM_HOSTINFO;
					if (!s.config->m_plugins_hostinfo) {
						platform = sinsp_plugin_platform::SINSP_PLATFORM_GENERIC;
					}
					inspector->open_plugin(cfg->m_name,
						                       cfg->m_open_params,
						                       platform);
```
or using the ternary operator
```suggestion
					inspector->open_plugin(cfg->m_name,
					                       cfg->m_open_params,
					                       s.config->m_plugins_hostinfo ? sinsp_plugin_platform::SINSP_PLATFORM_HOSTINFO : sinsp_plugin_platform::SINSP_PLATFORM_GENERIC);
```
We could even break the line and have both options on top of each other for compactness.

Reply:
Went with ternary operator, thanks!